app/models/manageiq/providers/vmware/infra_manager/orchestration_template.rb

```diff
@@ -28,6 +28,7 @@ def deployment_spec(opts)
 
     spec = {"accept_all_EULA" => opts[:accept_all_eula]}
     spec["name"] = opts[:vm_name] if opts[:vm_name].present?
+    spec["default_datastore_id"] = Storage.find_by(:id => opts[:storage_id]).ems_ref if opts[:storage_id].present?
```

Comment:
Should we be setting an ems_ref to an `_id` property?  Also if the `find_by` returns nil `.ems_ref` is going to blow up, recommend `.find_by()&.ems_ref`

Reply:
`default_datastore_id` is the name that VMware uses. MIQ saves it as `ems_ref` for a storage.